src/index.js

```diff
@@ -1,27 +1,74 @@
 import './style.css';
 
 const todoList = document.querySelector('.todo-list');
-const todos = [
-  {
-    index: 1,
-    completed: true,
-    task: 'Wash the dish',
-  },
-  {
-    index: 2,
-    completed: true,
-    task: 'Go to the gym',
-  },
-];
+const clearBtn = document.querySelector('.clear-btn');
+let todos = JSON.parse(localStorage.getItem('todos')) || [];
+
+function saveTasks() {
+  localStorage.setItem('todos', JSON.stringify(todos));
+}
+
+function removeTask(id) {
+  todos = todos.filter((task) => task.id !== id);
+  saveTasks();
+}
```

Comment:
According to the description for this milestone, upon the deletion of a task, the index for the remaining is expected to update to reflect the new array length. This will help make the editing functionality easy to implement.

To do this, when deleting a to-do, loop through the remaining todo and mutate the id field like so:

```
let newArr = []
remainingTodo.forEach((todo, id)=> {
  const task = {...todo}
  task.index = id
  
  newArr.push(task)
})

remainingTodo = newArr
```

DISCLAIMER: Test this and see if it helps, i haven't tested it, so you might encounter some bugs

[screen-capture (2).webm](https://github.com/H-Ramalan/Todo-List/assets/84930355/6b8de3cb-5627-42a9-948e-9f5c8cc827a0)